convex/importIntegrations.ts

```
@@ -0,0 +1,598 @@
+
+		// Verify user is a member of the workspace
+		const member = await ctx.db
+			.query("members")
+			.withIndex("by_workspace_id_user_id", (q) =>
+				q.eq("workspaceId", args.workspaceId).eq("userId", userId)
+			)
+			.unique();
+
+		if (!member) throw new Error("Not a member of this workspace");
+
+		let query = ctx.db
```

Biome (2.3.13)

[error] This let declares a variable that is only assigned once.

'query' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)